PresentScreenings/ListStreamer.cs

```diff
@@ -1,14 +1,15 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using Foundation;
 
 namespace PresentScreenings.TableView
 {
 	/// <summary>
     /// List streamer, read/write files based on list of derived objects.
     /// </summary>
 
-    public abstract class ListStreamer
+    public abstract class ListStreamer : NSObject
```

Comment:
Horrible.
Luckily, this can be undone when `Film` and `Screening` objects are no longer outlinable.

Reply:
Just added the user story to do that, #79!